tests/integration/standard/test_prepared_statements.py

```diff
@@ -153,7 +153,7 @@ def test_missing_primary_key_dicts(self):
     def _run_missing_primary_key_dicts(self, session):
         statement_to_prepare = """ INSERT INTO test3rf.test (v) VALUES  (?)"""
         # logic needed work with changes in CASSANDRA-6237
-        if self.cass_version[0] >= (3, 0, 0):
+        if self.cass_version[0] >= (2, 2, 8):
```

Comment:
why are you changing this ?

Reply:
I picked up these changes from python-driver-matrix, not sure we need them here